modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/60_wifi.js

```diff
@@ -185,6 +186,7 @@ return baseclass.extend({
 			this.callSessionAccess('access-group', 'luci-mod-status-index-wifi', 'read'),
 			this.callSessionAccess('access-group', 'luci-mod-status-index-wifi', 'write'),
 			L.hasSystemFeature('wifi') ? L.resolveDefault(uci.load('wireless')) : L.resolveDefault(),
+			firewall.getZones(),
```

Comment:
I think it's OK to put this above `L.has...` so you can keep data[x] contiguous.